### DISCUSSION TITLE
Setting up project in development mode

### DISCUSSION THREAD

**Discussion by Kodecheff** [unstructuredstudio/zubhub#571] [Q&A]
I have tried starting up the zubhub backend using `make start` as directed on the Readme but I'm not getting the expected results. Is there a way to make this process clearer and concise?

Also zubhub frontend development mode seems to run on node v16 and below. Might I suggest it be started also on the Readme file to ease set up for devs.

Thank you.

**Comment by Kodecheff** [comment]
> I have tried starting up the zubhub backend using `make start` as directed on the Readme but I'm not getting the expected results. Is there a way to make this process clearer and concise?
> 
> Also zubhub frontend development mode seems to run on node v16 and below. Might I suggest it be started also on the Readme file to ease set up for devs.
> 
> Thank you.

Here is the result I get from running `make start` - 
![image](https://user-images.githubusercontent.com/48069624/222162300-56dd124a-46f4-405f-8480-362abb509228.png)


Please how should I proceed to get the backend running?

**Reply by yaya-mamoudou** [reply]
In my case, I followed the documentation and didn't have any issues. 
After I ran make start, I went straight ahead to http://localhost:8000 and the API documentation was running.
Can you provide your next steps after running the `make start` command?

**Comment by psalmuelle** [comment]
@yaya-mamoudou is right. Your backend is running on this URL : [http://localhost:8000](http://localhost:8000).
You can also `make test` if you want to run tests.
As for the frontend issue, I have node v16.18.0 on my PC, so I didn't notice it couldn't run on higher versions. Did you encounter an error when your node version was greater than v16?
If there is an error, @tuxology, we might have to take a look at the issue!
